astropy/modeling/bounding_box.py

```diff
@@ -1,15 +1,15 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-
-
 """
 This module is to contain an improved bounding box.
 """
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
I thought the license comment is always above the module docstring (it is elsewhere)